### ISSUE TITLE
Server down?

### ISSUE THREAD

**Issue by Ithamar** [3mcd/javelin-fps#3]
It seems the demo link http://fps.javelin.games/ is not working anymore. If it is down for good, it might be a good idea to remove the link from both the GitHub page as well as from javelin.game ;)

**Comment by 3mcd** [comment]
Yes, thanks for pointing this out! I brought the FPS example down since it was difficult to maintain a sophisticated example like this while the API is experiencing so much churn. I'll remove it from the docs until I get it up to date with the latest version of Javelin, or produce a better multiplayer example. FWIW the @javelin/net lib is going to change a good bit soon with the introduction of a (hopefully) leaner/faster protocol, so it may take me a couple of weeks to get this up and running again.

In the meantime, this example works decently with the old API if you want to see one potential way of implementing these kinds of algorithms in your own game.

**Comment by Ithamar** [comment]
Cool, looking forward to all new things coming to javelin ;)
